test/gui/shared/scripts/pageObjects/AccountSetting.py

```diff
@@ -40,6 +40,13 @@ class AccountSetting:
         "type": "OCC::LogBrowser",
         "visible": 1,
     }
+    LOGOUT_OF_LOGIN_REQUIRED_DIALOG = {
```

Comment:
Is this `LOGOUT_OF_LOGIN_REQUIRED_DIALOG` already used or newly defined? It seems odd and cannot figure out the purpose by name.